cmd/onboard.go

```diff
@@ -0,0 +1,34 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/operate-first/opfcli/api"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdOnboard(opfapi *api.API) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "onboard file-path",
+		Long:  "The onboard command goes a step further than creating a project, and also creates the necesary groups, rolebindings, and adds users to the namespaces for a project.",
```

Comment:
I would rephrase the `Long` description to something like: 

`Onboard a project to a specific cluster in Operate First using a provided onboard config file.`